distribution/ddf-common/src/main/resources/etc/users.properties

```diff
@@ -32,5 +32,5 @@
 
 # DDF Changes: changed username/pw to admin/admin, added localhost certificate user
 
-admin=admin,group,admin,manager,viewer,webconsole
-localhost=localhost,group,admin,manager,viewer,webconsole
\ No newline at end of file
+admin=admin,group,admin,manager,viewer,webconsole,system-admin
```

Comment:
Dang it! You are right... teaches me to read the comments above a little more carefully.